src/System.Private.CoreLib/shared/Internal/Runtime/Augments/RuntimeThread.cs

```diff
@@ -1,56 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this remove feels out of place - was this intentional?

Reply:
It was removed from master and is just propagating the change back.

Reply:
just noticed as well after seeing several more files